tests/test_message/event_sample.py

```diff
@@ -0,0 +1,204 @@
+
+request = {
+  "ref": "refs/heads/main",
+  "before": "034c31e8e46431711bed9eada103aa552aaee6dd",
+  "after": "443f68ce112ab055bbf48666762a626baa4729f6",
+  "repository": {
+    "id": 807808293,
+    "node_id": "R_kgDOMCYtJQ",
+    "name": "demo_repo",
+    "full_name": "brngylni/demo_repo",
```

Comment:
Is there a way to emulate this without using a secondary external repository?

I want our test cases to be self-sufficient and not dependent on external factors.